lnbits/cache.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+import asyncio
+from time import time
+from typing import Any, NamedTuple, Optional
+
+from loguru import logger
+
+
+class Cached(NamedTuple):
+    value: Any
+    expiry: float
+
+
+def _add_prefix(key: str, prefix: str):
+    return prefix + ":" + key if prefix else key
+
+
+class Cache:
+    """
+    Small caching utility providing simple get/set interface (very much like redis)
+    """
+
+    def __init__(self):
+        self._values: dict[Any, Cached] = {}
+
+    def get(self, key: str, prefix: str = "") -> Optional[Any]:
```

Comment:
why no `defaultValue` (like `pop()`)

Reply:
The cache is an unordered key-value store... what would you expect with an argument-less `pop()`?